pkg/fnv/hash.go

```diff
@@ -0,0 +1,20 @@
+package fnv
+
+import (
+	"fmt"
+	"hash"
+	"hash/fnv"
+	"strconv"
+)
+
+// Hash hashes the given text using a 64-bit FNV-1a hash.Hash.
+func Hash(text string) (string, error) {
```

Comment:
These hash changes are related to the comment [0] in the refactoring PR but I think they are out of scope for this linting PR. Let's move this into it's separate pull request so we can keep the PR as one cohesive unit of work centered around linting.

[0] https://github.com/openshift/telemeter/pull/47#discussion_r225924456

Reply:
oki, agreed